### PR TITLE
#6337 Missing supplement

### PR DESCRIPTION
**Pull request by pklloveyou**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 
Missing a write to 'conditionlotid'

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by FrancescoBorzi** [comment]
@pklloveyou thanks for this PR. Can you please specify whether you tested this in-game and what problem this solves?

**Comment by pklloveyou** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/6337 A missing supplement

File: src/server/game/Loot/LootItemStorage.cpp [hunk 1]
```diff
@@ -95,6 +95,7 @@ void LootItemStorage::AddNewStoredLoot(Loot* loot, Player* /*player*/)
         stmt->setBool(index++, false);
         stmt->setBool(index++, false);
         stmt->setBool(index++, false);
+        stmt->setUInt32(index++, 0);
```
Comment (pklloveyou) [inline src/server/game/Loot/LootItemStorage.cpp:98]:
```
PrepareStatement(CHAR_INS_ITEMCONTAINER_SINGLE_ITEM, "INSERT INTO item_loot_storage (containerGUID, itemid, count, randomPropertyId, randomSuffix, follow_loot_rules, freeforall, is_blocked, is_counted, is_underthreshold, needs_quest, conditionLootId) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", CONNECTION_ASYNC);
```

Comment (pklloveyou) [reply src/server/game/Loot/LootItemStorage.cpp:98]: Missing a write to 'conditionlotid'

Comment (pklloveyou) [reply src/server/game/Loot/LootItemStorage.cpp:98]: @FrancescoBorzi

**Comment by pklloveyou** [comment]
I don't know how to test, but sometimes the console will report an error before PR, and the error will disappear after pr

**Comment by FrancescoBorzi** [comment]
> I don't know how to test, but sometimes the console will report an error before PR, and the error will disappear after pr

@pklloveyou ok, have you been running this with players?

**Comment by pklloveyou** [comment]
@FrancescoBorzi Yes, it has been used for a long time

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
> @FrancescoBorzi Yes, it has been used for a long time

fair enough, thank you!

**Pull request merged**